packages/constants/src/payload/payload.constant.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_MEMO_TYPE = 'Description';
```

Comment:
This is used only in the "extension" package, let's move it within `/gemwallet-extension/packages/extension/src/constants`, could you please add an end of line?

Reply:
Done [here](https://github.com/GemWallet/gemwallet-extension/pull/144/commits/0396971632122f29b0104e455a6a9c49ea59a336)